controllers/newsletter.js

```diff
@@ -0,0 +1,48 @@
+const newsletters = require("../models/newsletter");
+
+const ERROR_MESSAGE = "Something went wrong. Please try again or contact admin";
+
+const addEmail = async (req, res) => {
+  try {
+    const response = await newsletters.subscribe(req.body);
+    return res.json({
+      message: "Thank you for subscribing RDS Newsletter!",
+      response: response,
```

Comment:
Here both message and response convey similar message right....
Is this the right way to proceed with?

Reply:
any suggestions?